network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/configuration/NetpRequestInterceptor.kt

```diff
@@ -54,11 +63,23 @@ class NetpWaitlistRequestInterceptor @Inject constructor(
                     value = BuildConfig.NETP_DEBUG_SERVER_TOKEN,
                 )
             }
+
+            chain.proceed(
+                newRequest.build().also { logcat { "headers: ${it.headers}" } },
+            ).also {
+                networkProtectionRepository.vpnAccessRevoked = it.code == 403
```

Comment:
Would be good to re-confirm that 403 is only returned when header is ddg: bearer. Since if not, this could trigger the privacy pro stuff in prod.

Reply:
or we can add if `(subscriptions.isEnabled()) ` here too

Reply:
Adding `(subscriptions.isEnabled())` is better yeah